catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -463,12 +460,11 @@ private void classLoaderAndBundleContextSetup(BundleContext bundleContext) {
     ClassLoader tccl = Thread.currentThread().getContextClassLoader();
     try (InputStream stream = TikaMetadataExtractor.class.getResourceAsStream("/metadata.xslt")) {
       Thread.currentThread().setContextClassLoader(getClass().getClassLoader());
-      templates =
-          XMLUtils.getInstance()
-              .getSecureXmlTransformerFactory(
-                  net.sf.saxon.TransformerFactoryImpl.class.getName(),
-                  net.sf.saxon.TransformerFactoryImpl.class.getClassLoader())
-              .newTemplates(new StreamSource(stream));
+      XMLUtils.getInstance()
```

Comment:
Could use review here for sure. The templates variable didn't seem to be used, so not sure we need to be doing anything in here anymore

Reply:
I agree it doesn't look like `templates` was getting used. Unless the remaining code here (`XmlUtils.getInstance().get.....`) is doing something behind the scenes I don't think its necessary. It doesn't seem like `.newTemplates()` is doing anything important behind the scenes.

Reply:
Removed templates and the associated code around it.